### PR TITLE
Add cache-control and strict-transport-security headers

### PR DESCRIPTION
**Pull request by catchysong**

### CODE REVIEW

File: Caddyfile [hunk 1]
```diff
@@ -3,9 +3,12 @@
 reverse_proxy /api* {
   to {$BACKEND_URI}
   header_up Host {http.reverse_proxy.upstream.host}
+  header_up Cache-Control no-cache, no-store
+  header_up Strict-Transport-Security: max-age=31536000; includeSubDomains
```
Comment (markrawls) [inline Caddyfile:7]: `header_up` is for headers sent to the server, `header_down` is for requests sent to the browser. I think you've got them mixed up for everything except the Cache-Control directive

File: Caddyfile [hunk 2]
```diff
@@ -3,9 +3,12 @@
 reverse_proxy /api* {
   to {$BACKEND_URI}
   header_up Host {http.reverse_proxy.upstream.host}
+  header_up Cache-Control no-cache, no-store
```
Comment (markrawls) [inline Caddyfile:6]: I'm not 100%, but I think values with spaces in them need to be quoted.

**Review by markrawls — CHANGES_REQUESTED** [review]

**Review by ambikads — APPROVED** [review]

File: Caddyfile [hunk 3]
```diff
@@ -3,13 +3,15 @@
...
   header_up Host {http.reverse_proxy.upstream.host}
+  header_up Cache-Control "no-cache, no-store"
   header_down X-Content-Type-Options "nosniff"
   header_down X-XSS-Protection "1; mode=block"
 }
 
 reverse_proxy localhost:3000 {
   header_down X-FRAME-Options "DENY"
-  header_down Content-Security-Policy "frame-ancestors 'none'"
+  header_down Cache-Control "no-cache, no-store"
```
Comment (markrawls) [inline Caddyfile:13]: This I disagree with - we want people to cache the large blobs from the frontend, everything there is publicly accessible and it helps with performance. Other than that, LGTM.

**Review by markrawls — CHANGES_REQUESTED** [review]

File: Caddyfile (hunk 3 above)
Comment (catchysong) [reply Caddyfile]: Cool I just changed it back to frame ancestors 'none'

**Review by markrawls — APPROVED** [review]

**Pull request merged**